state/migration_import.go

```diff
@@ -841,6 +841,8 @@ func (i *importer) makeAddress(addr description.Address) address {
 		Scope:       addr.Scope(),
 		Origin:      addr.Origin(),
 		SpaceID:     addr.SpaceID(),
+		// todo(gfouillet): Update juju/description to support CIDR in Address interface
```

Comment:
Does this means migrations are broken until we do?

Reply:
I don't think so since our case is a very niche one, and CIDR is used after detecting some changes in local address, as far as model it in my head. 

However, for completness sake, I will push a PR on description. I just forgot this todo during sprint week. Thank for the reminder.

Reply:
@nvinuesa @manadart : I added the CIDR on description:
https://github.com/juju/description/pull/158

However to fix that one we will need to upgrade description from v5 to v7 in juju 3.5

Is it OK ?

Reply:
After discussion it appears that above PR won't help since it wouldn't be possible to upgrade juju/description in 3.5.

However, it is not required, because migration don't require to move CIDR. 

Moreover, in case of enable-ha, addresses seems to be refreshed from the machine, not from the DB, so if the CIDR is not exported/imported, it is not an issue.

I run some more tests by upgrading the controller to be sure:

```sh
juju upgrade-controller --build-agent --ignore-agent-versions
```

The bug isn't triggered after the upgrade.

I wonder if I should remove the comment or give some context (CIDR is required for enable-ha addresse space resolution, but no necessary yet to be migrated. Something like that.)

Reply:
If you QAd it and it works then that's fine by me (just add those steps to the QA of this PR to keep track of it). I'd leave a comment indicating that CIDR is only there to fix the linked bug and they are not needed to be migrated.

Reply:
I updated the comment, removing the todo and adding more context.